web/react/stores/socket_store.jsx

```diff
@@ -286,17 +281,12 @@ function handleChannelViewedEvent(msg) {
 
 var SocketStore = new SocketStoreClass();
 
-SocketStore.dispatchToken = AppDispatcher.register((payload) => {
+/*SocketStore.dispatchToken = AppDispatcher.register((payload) => {
     var action = payload.action;
 
     switch (action.type) {
-    case ActionTypes.RECIEVED_MSG:
-        SocketStore.handleMessage(action.msg);
-        SocketStore.emitChange(action.msg);
-        break;
-
     default:
     }
-});
+    });*/
```

Comment:
Because it was only using the AppDispatcher to talk to itself. The dispatcher is for global events.